openstack_exporter/__init__.py

```diff
@@ -0,0 +1,4 @@
+from pbr import version
```

Comment:
I guess this would need to be added to requirements, wouldn't it?

Reply:
pbr is in the requirements.txt currently.